src/Workspaces/Core/Portable/Shared/Extensions/SymbolInfoExtensions.cs

```diff
@@ -31,7 +31,9 @@ private static IEnumerable<ISymbol> GetAllSymbolsWorker(this SymbolInfo info)
 
         public static ISymbol GetAnySymbol(this SymbolInfo info)
         {
-            return info.GetAllSymbols().FirstOrDefault();
+            return info.Symbol != null
```

Comment:
This changes the return value to be a `Boolean` not an `ISymbol`.
As what it the `info.Symbol == null`